amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp

```diff
@@ -0,0 +1,258 @@
+/*
+Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+
+struct NormalizeLocalData {
+    vxRppHandle *handle;
+    Rpp32u deviceType;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    Rpp32s axisMask;
+    Rpp32f mean;
+    Rpp32f stdDev;
+    Rpp32f scale;
+    Rpp32f shift;
+    Rpp32f epsilon;
+    Rpp32s ddof;
+    Rpp32u numOfDims;
+    RpptDescPtr pSrcDesc;
```

Comment:
This file is still using the old normalize. The current Normalize will expect a generic desc ptr and generic ROI
Please check the Deepcam integration for Normalize and make changes in the Deepcam case, to handle for audio also if not already handled

This is the latest normalize call
https://github.com/snehaa8/rpp/blob/sn/normalize_ND/utilities/test_suite/HOST/Tensor_misc_host.cpp#L160

I will review this file, once it made compatible with latest normalize

Reply:
Please address this